.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
   global:
     # Set defaults to avoid repeating in most cases
     - ASTROCONDA=http://ssb.stsci.edu/astroconda
-    - CONDA_DEPS="cfitsio curl gcc pkg-config"
+    - CONDA_DEPS="cfitsio curl pkg-config"
```

Comment:
You could split this out into two, e.g. ``ASTROCONDA_DEPS`` and ``CONDA_FORGE_DEPS``.